### PR TITLE
Hypernoblium formation no longer stopped by Hypernoblium oppression

### PR DESCRIPTION
**Pull request by ghost**
# Intent of your Pull Request

No longer do atmosians have to suffer 95% yield loss because of a design oversight! No longer do you have to spend 30 minutes reheating and recooling your loop to get 1% more yield of hypernoblium! 

Literally just changes two numbers to change the reaction order.

#### Changelog

:cl:  
tweak: Hypernoblium formation no longer stopped by Hypernoblium oppression.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
I haven't checked if setting both to priority INFINITY would work but I assume that would just cause wonkiness. 1000 and 1001 should be fine unless we get literally a thousand gas reactions.

**Comment by Takahiru** [comment]
yes

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	return NO_REACTION
 
 /datum/gas_reaction/nobliumsupression
-	priority = INFINITY
+	priority = 1000 //just make sure everything else except nobformation is lower than 1000 and we're good. Can't set to INF because that'd fuck over nob formation.
```
Comment (JamieD1) [inline code/modules/atmospherics/gasmixtures/reactions.dm]:
```suggestion
	priority = 1000
```

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 2]
```diff
@@ -398,7 +398,7 @@
 		return REACTING
 
 /datum/gas_reaction/nobliumformation //Hyper-Noblium formation is extrememly endothermic, but requires high temperatures to start. Due to its high mass, hyper-nobelium uses large amounts of nitrogen and tritium. BZ can be used as a catalyst to make it less endothermic.
-	priority = 6
+	priority = 1001 //1 higher than nob stop because I can't really set this to infinity.
```
Comment (JamieD1) [inline code/modules/atmospherics/gasmixtures/reactions.dm]:
```suggestion
	priority = 1001
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

**Comment by JamieD1** [comment]
Either make the comments less or remove them

**Review by JamieD1 — APPROVED** [review]

**Comment by ShoziOffical** [comment]
DIDNT WORK SMH!!!!

**Comment by Rdrazga** [comment]
Bruh iv been waiting for years for a hyper noblium formation fix and according to warface it may not have even worked, feels bad.

**Comment by ShoziOffical** [comment]
> Bruh iv been waiting for years for a hyper noblium formation fix and according to warface it may not have even worked, feels bad.

i made a fix pr

**Pull request merged**